app/src/main/java/com/github/se/cyrcle/model/parking/ParkingRepositoryFirestore.kt

```diff
@@ -60,15 +62,15 @@ class ParkingRepositoryFirestore(private val db: FirebaseFirestore) : ParkingRep
       onSuccess: (List<Parking>) -> Unit,
       onFailure: (Exception) -> Unit
   ) {
-    if (start.latitude > end.latitude || start.longitude > end.longitude) {
+    if (start.latitude() > end.latitude() || start.longitude() > end.longitude()) {
```

Comment:
Here i would just rename the arguments so that it's easier to understand what they refer to. For example `bottomLeftCorner` and `topRightCorner`. However it doesn't remove my approval good job